### PR TITLE
Feature api updates

### PR DESCRIPTION
**Pull request by petterarvidsson**

### CODE REVIEW

**Comment by petterarvidsson** [comment]
@nsg I am ready to merge this one now. What do you think?

**Comment by nsg** [comment]
@petterarvidsson maybe rebase and kill commit 92ef814, 5c4b87d and  1521fe9?

**Comment by petterarvidsson** [comment]
Sure!

File: src/main/scala/konstructs/konstructing.scala [hunk 1]
```diff
@@ -19,6 +19,8 @@ class KonstructingActor(konstructs: Set[Konstruct], konstructedFilters: Seq[Acto
 
   def receive = {
     case MatchPattern(pattern: Pattern) =>
+      println(pattern)
+      println(konstructs)
```
Comment (nsg) [inline src/main/scala/konstructs/konstructing.scala]: Debug lines I assume? Remove these

Comment (petterarvidsson) [inline src/main/scala/konstructs/konstructing.scala]: Indeed :-)

**Comment by nsg** [comment]
Looks good :+1: 

@petterarvidsson

**Pull request merged**